tox.ini

```diff
@@ -1,11 +1,10 @@
 [tox]
 ; envlist = {py37,py38,py39}-sanic{21.6}, check
-envlist = {py37,py38,py39}, check
```

Comment:
Why is Py37 removed? Are we going to remove that in Sanic app for upcoming release too?

Reply:
It was never supported in this lib and should not have been here to begin with. I think the main lib continues to support 3.7 as long as it is officially supported.